test/unit/android-helper-specs.js

```diff
@@ -81,6 +81,16 @@ describe('Android Helpers', () => {
       await helpers.prepareEmulator(adb, {}).should.eventually.be.rejected;
     });
   }));
+  describe('ensureNetworkSpeed', () => {
+    it('should return value if network speed is valid', async () => {
+      adb.NETWORK_SPEED = { GSM: 'gsm' };
```

Comment:
per our project's style, we don't put spaces around braces unless they're imports/exports. so this (and throughout) would be like:

```js
adb.NETWORK_SPEED = {GSM: 'gsm'};
```